docs/api/kustomize.md

```diff
@@ -2,16 +2,16 @@
 <p>Packages:</p>
 <ul class="simple">
 <li>
-<a href="#kustomize.toolkit.fluxcd.io%2fv1beta2">kustomize.toolkit.fluxcd.io/v1beta2</a>
+<a href="#kustomize.toolkit.fluxcd.io%2fv1">kustomize.toolkit.fluxcd.io/v1</a>
```

Comment:
To align the structure within docs/api/ with NC and SC we should move this file (docs/api/kustomize.md) into docs/api/v1/ (I know it's only one API version at this point but it makes it easier for contributors to navigate this directory structure when it's the same across all Flux repos).

Reply:
I have updated the docs and moved the generated ones to `/v1`, please make these changes in the website PR to verify all is good.